test_framework/terraform/aws/sles/main.tf

```diff
@@ -256,3 +256,16 @@ resource "aws_eip" "lh_aws_eip_controlplane" {
   count    = var.lh_aws_instance_count_controlplane
   vpc      = true
 }
+
+resource "aws_ebs_volume" "lh_aws_hdd_volume" {
+
+  count = var.use_hdd ? var.lh_aws_instance_count_worker : 0
+
+  availability_zone = var.aws_availability_zone
+  size              = 160
+  type              = "st1"
```

Comment:
ref: `Throughput Optimized HDD ( st1 ) volumes provide low-cost magnetic storage that defines performance in terms of throughput rather than IOPS.`